network/network_windows_test.go

```diff
@@ -227,3 +227,28 @@ func TestDeleteNetworkImplHnsV1WithTimeout(t *testing.T) {
 		t.Fatal("Failed to timeout HNS calls for deleting network")
 	}
 }
+
+func TestAddIPv6DefaultRoute(t *testing.T) {
+	nm := &networkManager{
+		ExternalInterfaces: map[string]*externalInterface{},
+	}
+
+	nwInfo := &NetworkInfo{
+		Id:           "d3f97a83-ba4c-45d5-ba88-dc56757ece28",
+		MasterIfName: "eth0",
+		Mode:         "bridge",
+	}
+
+	extInterface := &externalInterface{
+		Name:    "eth0",
+		Subnets: []string{"ipv4Subnet", "ipv6Subnet"},
+	}
+
+	Hnsv2 = hnswrapper.NewHnsv2wrapperFake()
+
+	_, err := nm.newNetworkImplHnsV2(nwInfo, extInterface)
+	if err != nil {
+		fmt.Printf("+%v", err)
+		t.Fatal(err)
+	}
+}
```

Comment:
can you add for cases for:
1.  where hns network creation succeed and route failed and on second attempt it should succeed without creating hns network again?
2. failure cases for each error returned and if error returned is expected

Reply:
1. Yes, once hns network is created successfully, then it will attempt to execution command to add ipv6 default route; it will try 3 times to add default route; if they are all failed, just log error and it won't create hnsNetwork again;